src/kbmod_wf/task_impls/reproject_single_chip_single_night_wu.py

```diff
@@ -0,0 +1,96 @@
+from kbmod.work_unit import WorkUnit
+
+import kbmod.reprojection as reprojection
+
+from astropy.reproject.mosaicking import find_optimal_celestial_wcs
+import os
+import time
+from logging import Logger
+
+
+def reproject_wu(
+    original_wu_filepath: str = None,
+    reprojected_wu_filepath: str = None,
+    runtime_config: dict = {},
+    logger: Logger = None,
+):
+    """This task will perform reflex correction and reproject a WorkUnit to a common WCS.
```

Comment:
Nit: this no longer performs reflex correction

Reply:
Fixed